### PR TITLE
adding enrichment_swe_capacity parameter for practice purpose

### PR DESCRIPTION
**Pull request by asmfakhrul**

### CODE REVIEW

File: cruiser/scenarios/eg01_eg23.py [hunk 1]
```diff
@@ -22,6 +22,10 @@ def mine_throughput(self, value):
     def enrichment_tails_assay(self, value):
         self.sim['simulation']['facility'][1]['config']['Enrichment']['tails_assay'] = value
 
+    @inparam(default=1E100, widget='FloatSlider', min=0.0, max=0.008, step=0.0001, readout_format='.4f')
```
Comment (scopatz) [inline cruiser/scenarios/eg01_eg23.py]: The default value here is greater than the max value.  That seems wrong....

**Comment by scopatz** [comment]
Thanks for putting this in @asmfakhrul !

**Comment by asmfakhrul** [comment]
Dr. Scopatz, please review the new pull request.

**Comment by scopatz** [comment]
Looks good thanks!

**Pull request merged**